frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -206,57 +193,64 @@ export default class Select extends Component {
       .filter(n => n);
 
     return (
-      <PopoverWithTrigger
-        ref={ref => (this._popover = ref)}
-        triggerElement={
-          this.props.triggerElement || (
-            <SelectButton
-              ref={this.selectButtonRef}
-              className="flex-full"
-              hasValue={selectedNames.length > 0}
-              disabled={disabled}
-              {...buttonProps}
-            >
-              {selectedNames.length > 0
-                ? selectedNames.map((name, index) => (
-                    <span key={index}>
-                      {name}
-                      {index < selectedNames.length - 1 ? ", " : ""}
-                    </span>
-                  ))
-                : placeholder}
-            </SelectButton>
-          )
-        }
-        onClose={composeEventHandlers(onClose, this.handleClose)}
```

Comment:
Since I dropped `handleClose` I only pass `onClose` to `onHide` as `TippyPopoverWithTrigger` doesn't expose `onClose` on purpose, but I tested that `onHide` works as expected i.e. `onClose` would be call any time the popover is closed.